sdk/Pulumi/Deployment/Deployment.cs

```diff
@@ -226,9 +226,9 @@ internal Task<bool> MonitorSupportsDeletedWith()
         /// Returns whether the resource monitor we are connected to supports the "aliasSpec" feature across the RPC interface.
         /// In which case we no longer compute alias combinations ourselves but instead delegate the work to the engine.
         /// </summary>
-        internal Task<bool> MonitorSupportsAliasSpec()
+        internal Task<bool> MonitorSupportsAliasSpecs()
         {
-            return MonitorSupportsFeature("aliasSpec");
+            return MonitorSupportsFeature("aliasSpecs");
```

Comment:
oh well this is sad
I'll raise a PR for the protobuf to say we should stop adding more strings here and generate an enum for engine features instead. I don't think we want to try moving the existing flags, but if we add anymore it'll be good to see a note reminding us about this.

Reply:
An enum would be great for this